suricata.yaml.in

```diff
@@ -983,6 +983,13 @@ asn1-max-frames: 256
 # umask passed on by the shell.
 #umask: 022
 
+# Global memcap
+# This value represents the maximum amount of memory that can be used
+# to allocate all the memcap values.
+# The sum of all of them (stream.memcap, flow.memcap, etc.) cannot be
+# greater than the global memcap.
```

Comment:
Suggest adding "By default, the global memcap is disabled. Set a value to enable".